cypress/e2e/models/migration/migration-waves/migration-wave.ts

```diff
@@ -276,4 +276,47 @@ export class MigrationWave {
         cy.contains("li", date.toLocaleString("en-us", { month: "long" })).click();
         cy.contains("button:not([disabled])", `${date.getDate()}`).first().click();
     }
+
+    public clickWaveStatus() {
+        cy.contains(this.name)
+            .parents("tr")
+            .within(() => {
+                cy.get(MigrationWaveView.waveStatusColumn).click();
+            });
+    }
+
+    public isExpanded(): Cypress.Chainable<boolean> {
+        return cy
+            .contains(this.name)
+            .parent()
+            .parent()
+            .then(($element) => {
+                return $element.hasClass(MigrationWaveView.waveExpanded);
+            });
+    }
+
+    public removeApplication(applicationName) {
```

Comment:
one last comment , this method should accept a list of applications to be able to remove all of them one by one.